pages/index.js

```diff
@@ -5,28 +5,52 @@ import Image from "next/image";
 import TripInputForm from "@/components/TripInputForm";
 import dynamic from "next/dynamic";
 import { sortTrips, countdown } from "@/components/utils";
+import destinations from "@/db/destinations";
+import { useState } from "react";
+import { format } from "date-fns";
 
 const DynamicCarousel = dynamic(() => import("../components/Carousel"), {
   ssr: false,
 });
 
 export default function Homepage({ trips, setTrips }) {
+  console.log(new Date());
+  const [startDate, setStartDate] = useState();
+  const [endDate, setEndDate] = useState();
+  const [successMessage, setSuccessMessage] = useState("");
+  function handleChange(dates) {
+    const [start, end] = dates;
+    setStartDate(start);
+    setEndDate(end);
+  }
   const { currentTrips, upcomingTrips } = sortTrips(trips);
 
   function handleAddTrip(event) {
+    if (!startDate || !endDate) {
+      console.log("error");
```

Comment:
As this is not helpful for the user to see what went wrong (as user usually do not check the console) it may be better to implement a  message for the user with information what the user can do next to prevent.